### PR TITLE
882 all platforms leavechannelex updated

### PR DESCRIPTION
**Pull request by hussain-khalid**

### CODE REVIEW

**Comment by billy-the-fish** [comment]
Can you sort out the conflicts and sort out the DRM changes you have in here.

File: shared/extensions-marketplace/drm-play/project-test/electron.mdx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<PlatformWrapper platform = "electron">
+
+3.  In **preload.js**, update `appID`, `channel` and `token` with your values.
```
Comment (atovpeko) [inline shared/extensions-marketplace/drm-play/project-test/electron.mdx:3]:
```suggestion
3.  In **preload.js**, update `appID`, `channel`, and `token` with your values.
```

File: shared/video-sdk/develop/ensure-channel-quality/project-implementation/electron.mdx [hunk 2]
```diff
@@ -218,8 +223,34 @@ To implement the call quality features, take the following steps:
     }
     
     ```
+1. **Configure the <Vpd k="SDK" /> log file**
+
+    To customize the location, content and size of log files, in `preload.js`, add the following code to `window.onload()` before `agoraEngine.registerEventHandler(EventHandles);`:
```
Comment (atovpeko) [inline shared/video-sdk/develop/ensure-channel-quality/project-implementation/electron.mdx:228]:
```suggestion
    To customize the location, content, and size of log files, in `preload.js` add the following code to `window.onload()` before `agoraEngine.registerEventHandler(EventHandles);`:
```

File: shared/video-sdk/develop/ensure-channel-quality/project-implementation/electron.mdx [hunk 3]
```diff
@@ -218,8 +223,34 @@ To implement the call quality features, take the following steps:
...
+    agoraEngine.setLogLevel(LogLevel.LogLevelWarn);
+    ```
+    Make sure you replace `<user_name>` from the file path string with the user name of your development device.
+
+    If you want to upload the log file automatically to a CDN, call `setLocalAccessPoint(LocalAccessPointConfiguration config)` to specify the local access point and assign the native access module to the SDK.
+
+<ProductWrapper product="interactive-live-streaming">
+5. **Set the latency level**
+
+    To specify the audience latency level, in `preload.js`, add the following code after `if (document.getElementById('host').checked) {`:
```
Comment (atovpeko) [inline shared/video-sdk/develop/ensure-channel-quality/project-implementation/electron.mdx:243]:
```suggestion
    To specify the audience latency level, in `preload.js` add the following code after `if (document.getElementById('host').checked) {`:
```

**Review by atovpeko — CHANGES_REQUESTED** [review]
Just a few language things

**Pull request merged**